project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject puppetlabs/lein-ezbake "1.2.2-SNAPSHOT"
+(defproject puppetlabs/lein-ezbake "1.3.0-SNAPSHOT"
```

Comment:
@shrug Should we do this as a Y or Z bump?  Looks like you've been doing Z bumps for a lot of releases that have features in them.  We're used to doing Y bumps for new functionality in other projects (even without breaking changes).  Wanted to see if that works with your current versioning strategy.